R/process-KNMI-netCDF-files.R

```diff
@@ -0,0 +1,100 @@
+# Function to process individual netCDF files ####
+
+# Author: Cherine Jantzen
+# Created: 2024-02-21
+# Last updated: 2024-02-22
+
+# I. Load packages --------------------------------------------------------
+
+library(tidync)
+library(ncmeta)
+library(RNetCDF)
+library(dplyr)
+library(purrr)
+library(tidyr)
+library(here)
+
+# II. Function ------------------------------------------------------------
+
+# Arguments: 
+# - bbox: Spatial bounding box for which to retrieve data. Vector of four numeric values, indicating western-most, southern-most, eastern-most and northern-most point of the bounding box (in Amersfoort/RD New (ESPG:28992)).
+# - netCDF_location: Character specifying the folder name in which the netCDF files are stored. Folder needs to be a sub-folder of the working directory. 
+
+process_netCDF <- function(bbox,
```

Comment:
Should we add a reference to KNMI in the name of this function? As this function would only be able to process netCDF files that have the structure that the KNMI netCDF files have, right?